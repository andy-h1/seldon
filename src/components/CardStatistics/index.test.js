import React from 'react';
import { render } from '@testing-library/react';
import { CardStatistics } from '.';

const title = 'Project Views';
const statistic = 25670;

describe('Statistics Card component', () => {
  it('should render', () => {
    const { container } = render(<CardStatistics title={title} statistic={statistic} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
