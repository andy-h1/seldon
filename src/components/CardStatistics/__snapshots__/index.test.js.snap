// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statistics Card component should render 1`] = `
<div
  class="sc-bdfBwQ ezRtVh"
>
  <p
    class="sc-gsTCUz jPeWXq"
  >
    Project Views
  </p>
  <p
    class="sc-dlfnbm bfXIvc"
  >
    25670
  </p>
</div>
`;
